tests/e2e/Adapter/MongoDBTest.php

```
@@ -0,0 +1,108 @@
+
+    public function testUpdateAttributeStructure(): void
+    {
+        $this->assertTrue(true);
+    }
+
+    public function testKeywords(): void
+    {
+        $this->assertTrue(true);
+    }
+
+    protected static function deleteColumn(string $collection, string $column): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$collection'. (Unused Code Rules)

(UnusedFormalParameter)